tests/test_models.py

```diff
@@ -871,6 +871,38 @@ def test_string_conversion(self, method):
         )
         self.assertEqual(method(enterprise_catalog), expected_str)
 
+    @mock.patch('enterprise.utils.configuration_helpers')
+    def test_catalog_info_in_course_enrollment_url(self, config_mock):
+        faker = FakerFactory.create()
+        uuid = faker.uuid4()  # pylint: disable=no-member
+        name = 'EnterpriseWithACatalog'
+        expected_url = '/enterprise/14736f84-a29c-9685-bfaa-a9b5e0ecb9ac/course/course-v1:edX+DemoX+Demo_Course_1/' \
+                       'enroll/?utm_medium=enterprise&utm_source=enterprisewithacatalog&catalog='+uuid
+        enterprise_catalog = EnterpriseCustomerCatalog(
+            uuid=uuid,
+            enterprise_customer=EnterpriseCustomerFactory(uuid='14736f84-a29c-9685-bfaa-a9b5e0ecb9ac', name=name)
```

Comment:
Please use faker to generate the enterprise customer UUID and use it in constructing the `expected_course_enrollment_url` and creating the enterprise customer.